apps/mobile/src/features/receive/screens/select-account.tsx

```diff
@@ -12,7 +12,7 @@ type CurrentRoute = CreateCurrentReceiveRoute<'select-account'>;
 
 export function SelectAccount() {
   const navigation = useReceiveSheetNavigation<CurrentRoute>();
-  const accounts = useAccounts('active');
+  const accounts = useAccounts();
```

Comment:
@tigranpetrossian is this OK with you? I changed it so we infer `useAccounts` as `active`. 

Maybe it's better to:
- introduce `useActiveAccounts`
- instead of changing this update everywhere else to be `useAccounts('active'``

@edgarkhanzadian we use `useAccounts` a lot in `Approver`. I guess you only really want active accounts there also?